man/systemd.network.xml

```diff
@@ -1004,6 +1004,25 @@ DuplicateAddressDetection=none</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MulticastIGMPVersion=</varname></term>
+        <listitem>
+          <para>Configure IPv4 Multicast IGMP Version. If not configure not change the IPV4 Multicast IGMP version.
+          If configured it takes one of
+          <literal>no</literal>, <literal>1</literal>, <literal>2</literal>, or <literal>3</literal>.
+          When <literal>no</literal> no enforcement of a IGMP version, IGMPv1/v2 fallback allowed, will back to
+          IGMPv3 mode again if all IGMPv1/v2 Querier Present timer expires no source validation will be done.
+          When <literal>v1</literal> enforce to use IGMP version 1, will also reply IGMPv1 report if receive
+          IGMPv2/v3 query.
+          When <literal>v2</literal> enforce to use IGMP version 2, will fallback to IGMPv1 if receive IGMPv1 query
+          message. Will reply report if receive IGMPv3 query.
+          When <literal>v3</literal> enforce to use IGMP version 3, the same react with "no".
+          </para>
```

Comment:
hmm, these are not proper English sentences..

Reply:
updated